eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/implementation/ReactorSession.java

```diff
@@ -235,8 +237,7 @@ public boolean removeLink(String linkName) {
     }
 
     private ActiveClientTokenManager createTokenManager(String entityPath) {
-        final String tokenAudience = String.format(Locale.US, ClientConstants.TOKEN_AUDIENCE_FORMAT, sessionHandler.getHostname(), entityPath);
-        return new ActiveClientTokenManager(cbsNodeSupplier, tokenAudience, ClientConstants.TOKEN_VALIDITY, ClientConstants.TOKEN_REFRESH_INTERVAL);
+        final String tokenAudience = audienceProvider.getResourceString(entityPath);
```

Comment:
Here `audienceProvider` is an instance of `TokenResourceProvider` class right?
I worry, we will shoot ourselves in the foot with the whole audience vs resource terminology... sigh.

Something to think about for Preview 2